README.md

```diff
@@ -416,6 +416,66 @@ The local key helper is responsible for:
 - Producing signatures with the binding key
 - Cleanup of the binding key and its artifacts (when the user clears the browser session or the key is unused for a long time)
 
+From the deployment point of view there are two types of local key helpers: _well-known_ and _third party_.
+
+_Well-known local key helpers_ come with either OS or built into the browser. It assumed that well-known key helpers are **trusted and enabled by default** in a browser. A browser knows how to activate a well-known key helper.
```

Comment:
We should define how `browser will know how to activate a well-known key helper`

Reply:
Defined in the line 477.